<!-- Actor -->
<element name="actor" required="*">
  <attribute name="name" type="string" default="__default__" required="1"/>
  <attribute name="static" type="bool" default="false" required="0"/>

  <element name="origin" required="0">
    <attribute name="pose" type="pose" default="0 0 0 0 0 0" required="1"/>
  </element> <!-- End Origin -->

  <element name="skin" required="1">
    <attribute name="filename" type="string" default="__default__" required="1"/>
    <attribute name="scale" type="double" default="1.0" required="0"/> 
  </element> <!-- End Skin -->

  <element name="animation" required="+">
    <attribute name="name" type="string" default="__default__" required="1"/>
    <attribute name="filename" type="string" default="__default__" required="1"/>
    <attribute name="scale" type="double" default="1.0" required="0"/>
    <attribute name="interpolate_x" type="bool" default="false" required="0"/>
  </element> <!-- End Animation -->

  <element name="script" required="1">
    <attribute name="loop" type="bool" default="true" required="0"/>
    <attribute name="delay_start" type="double" default="0.0" required="0"/>
    <attribute name="auto_start" type="bool" default="true" required="0"/>
    <element name="trajectory" required="*">
      <attribute name="id" type="int" default="0" required="1"/>
      <attribute name="type" type="string" default="__default__" required="1"/>
      <element name="waypoint" required="*">
        <attribute name="time" type="double" default="0.0" required="1"/>
        <attribute name="pose" type="pose" default="0 0 0 0 0 0" required="1"/>
      </element> <!-- End Waypoint -->
    </element> <!-- End Action -->
  </element> <!-- End Script -->
  
  <include filename="link.sdf" required="+"/>
  <include filename="joint.sdf" required="*"/>
  <include filename="plugin.sdf" required="*"/>

</element> <!-- End Actor -->
